integration_test/deploy-dr-buckets.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+set -ex
+
+export AWS_ACCESS_KEY_ID=$bamboo_AWS_ACCESS_KEY_ID
+export AWS_SECRET_ACCESS_KEY=$bamboo_AWS_SECRET_ACCESS_KEY
+export AWS_DEFAULT_REGION=$bamboo_AWS_DEFAULT_REGION
+
+#remove old files from bamboo as they throw error
+rm *.tf
+git clone --branch develop --single-branch https://github.com/nasa/cumulus-orca.git && cd integration_test
+#replace prefix with bamboo prefix variable
+sed -e 's/PREFIX/'"$bamboo_PREFIX"'/g' dr-buckets.tf.template > dr-buckets.tf
+
+if ! aws s3 cp s3://$bamboo_PREFIX-dr-tf-state/dr-buckets-tf/terraform.tfstate .;then
+    echo "terraform state file not present in S3. Creating the buckets in DR account."
+else
+    echo "Terraform state file found in S3 bucket. Using S3 remote backend"
+fi
+
+terraform init -input=false
+echo "Deploying S3  buckets in Disaaster Recovery account"
+terraform apply \
+  -auto-approve \
+  -input=false
+
+#copy terraform state file to the created tf-state bucket
```

Comment:
Why do we make the copy? Shouldn't that be configured beforehand? If we have a chicken and egg problem maybe we should look at https://github.com/stavxyz/terraform-aws-backend to create our state bucket and dynamo table.

Reply:
we need a `prefix-tf-state` bucket created first before we can use that for configuring as backend. 

so first time when the script is run, it copies the state file to the created bucket to use as backend. 

I did go through the link you sent but found that somewhat confusing to use. Plus seems like they have some additional bucket properties we don't need and dynamodb properties are different https://github.com/stavxyz/terraform-aws-backend/blob/master/main.tf

The https://github.com/stavxyz/terraform-aws-backend/blob/beb7ee0b229701f47c9853fb30529dbc568da0e5/main.tf#L83 will also cause issue while deleting the resource later on

If we decide to create the tf-state bucket and dynamodb table manually first instead of automating, then we will not have this dependency.